pkg/mcs/exportspec_test.go

```diff
@@ -33,6 +33,25 @@ var (
 			},
 		},
 	}
+
+	different_name_sample = mcs.ExportSpec{
```

Comment:
Is this duplicating the global `sample` variable? Can you reuse it?

Reply:
Yes, its useless, I think its trace of the attempts before adding the mutate function (I deleted it now)